subsys/net/lib/lwm2m/Kconfig

```diff
@@ -597,6 +605,14 @@ config LWM2M_GATEWAY_IOT_DEVICE_OBJECTS_MAX_STR_SIZE
 
 endif # LWM2M_GATEWAY_OBJ_SUPPORT
 
+config LWM2M_BLOCK_TRANSFER_TESTS
```

Comment:
Ditto, if it's really needed to check if the code is executed in test, use `CONFIG_ZTEST` .